src/app/phast/losses/losses-tabs/energy-input-tab/energy-input-tab.component.ts

```diff
@@ -74,31 +75,39 @@ export class EnergyInputTabComponent implements OnInit {
       }
     }
   }
-  checkMissingData(): boolean {
-    let testVal = false;
+
+  checkLossData(): { invalid: boolean, hasWarning: boolean } {
+    let baselineValid = true;
+    let baselineWarning: boolean = false;
     if (this.energyInputCompareService.baselineEnergyInput) {
+      let baselineResults: PhastResults = this.phastResultsService.getResults(this.phast, this.settings);
       this.energyInputCompareService.baselineEnergyInput.forEach(loss => {
-        if (this.checkLossValid(loss, this.phast) === false) {
-          testVal = true;
-        }
```

Comment:
I think this logic needs to be double checked. It sill set the booleans to whatever the last item in the for loop is.

Same for modification.